R/commonMachineLearningClassification.R

```diff
@@ -352,9 +352,7 @@
   for (row in 2:l) {
     for (col in 1:(l-1)) {
       if (row == col) {
-          p <- jaspGraphs::drawAxis(xName = "", yName = "", force = TRUE) + adjMargin
-          p <- p + ggplot2::xlab("")
-          p <- p + ggplot2::ylab("")
+          p <- jaspGraphs::drawAxis(xName = NULL, yName = NULL, force = TRUE) + adjMargin
```

Comment:
It looks like those two statements can be combined, although note that due to
```r
  for (row in 2:l) {
    for (col in 1:(l-1)) {
```
we always have `row > col`, so maybe we can remove both the `if (row == col)` and `if (col > row)`?

Also, if you run `jaspGraphs::drawAxis` in Rstudio you'll see the message `"This function will be deprecated"`, so it's best to remove it wherever you encounter it. I'm not sure why `adjMargin` is there but it's probably better to leave it in:
```suggestion
                    p <- ggplot2::ggplot() + themeJaspRaw() + adjMargin
```